spec/system/qualifications/user_gives_feedback_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "Feedback", type: :system do
+  include CommonSteps
+  include QualificationAuthenticationSteps
+
+  scenario "User gives feedback", test: %i[with_stubbed_auth with_fake_quals_api] do
+    given_the_qualifications_service_is_open
+    and_i_am_signed_in_via_identity
+    and_i_click_on_feedback
+    then_i_see_the_feedback_form
+    when_i_press_send_feedback
+    then_i_see_validation_errors
+    when_i_choose_satisfied
+    when_i_fill_in_how_we_can_improve
+    when_i_choose_yes
+    when_i_enter_an_email
+    when_i_press_send_feedback
+    then_i_see_the_feedback_sent_page
```

Comment:
I would add a final step here (something like `and_my_feedback_is_saved`) which directly evaluates the present of a feedback record in the database, and checks that its `service` value is `aytq`.

A similar change should be made to the existing spec for Check feedback. 

This would have helped, for example, in catching bugs like https://github.com/DFE-Digital/access-your-teaching-qualifications/pull/636/files#r1582842956